target/linux/mvebu/cortexa9/base-files/etc/init.d/helios4_fancontrol

```diff
@@ -0,0 +1,97 @@
+#!/bin/sh /etc/rc.common
+
+# control pwm fans with basic 10-point fan speed increments
+
+START=98
+
+start() {
+	case $(board_name) in
+	kobol,helios4)
```

Comment:
I'm not sure whether the file should be named more generic, if devices are switched inside anyway.

Reply:
yeah will give it a more generic name